api/proto/attest/attest.proto

```diff
@@ -157,3 +157,8 @@ enum AttestStorageKeysResponseCode {
 message AttestStorageKeysResp {
   AttestStorageKeysResponseCode response = 1;    //Result of processing Z_ATTEST_REQ_TYPE_STORE_KEYS in Controller
 }
+
+message AttestVolumeKeyData {
+  bytes encrypted_key = 1; //key, in encrypted format
+  bytes public_key = 2;    //Zpoint, to recover the key
```

Comment:
Unclear what Zpoint is

Reply:
Hi @rvs, yes the comment needs correction - the public key of the ephemeral ECC key, which is required along with the ECDH private key in TPM,  to derive the shared secret, which will act as the seed to generate the AES decryption key to decrypt the encrypted key.

Reply:
Corrected it